src/components/CourseCard.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { far } from "@fortawesome/free-regular-svg-icons";
+import styles from "./courseStyles.module.css";
+
+library.add(fas, far);
+
+function CourseCard(props) {
+  let course = props.course;
+  return (
+    <div className={styles.course}>
+      <img
+        src={require("../" + course.course_img)}
+        alt={course.category + "Course"}
+      />
+      <h2>{course.title}</h2>
+      <span>{course.author}</span>
+      <br />
+      <span className={styles.rating}>{course.rating} </span>
+
+      <FontAwesomeIcon
```

Comment:
instead of duplicate the FontAwesomeIcon multiple time you can use loop or your can create another component " FontAwesomeIconList " which accept a number as props and inside it to the loop like ( <FontAwesomeIconList count={5} />